frontend/packages/client/src/components/ProposalsList/ProposalCardHeader.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { Star } from "components/Svg";
+import Blockies from "react-blockies";
+import { useUserRoleOnCommunity } from "hooks";
+import { useParams } from "react-router-dom";
+
+const ProposalCardHeader = ({ creatorAddr }) => {
+  const { communityId } = useParams();
+  const isAdmin = useUserRoleOnCommunity({
+    creatorAddr,
+    communityId,
+    roles: ["admin"],
+  });
```

Comment:
I think this might not be efficient, and we can work it on a separate PR and [ticket](https://linear.app/dappercollectives/issue/CAS-88/improve-use-of-useuserroleoncommunity-for-proposal-list), but the userRoleOnCommunity is using the endpoint `/users/${addr}/communities?count=${count}&start=${start}` which will bring all the communities that the address is part of, then we are making the same request for all headers. I think we can move this out of the proposal card header, and pass it down as a prop. The request would be done inside `useCommunityProposalWithVotes` [here](https://github.com/DapperCollectives/CAST/blob/fb64ffde79e37522cf555c3deb98a2831988126c/frontend/packages/client/src/hooks/useCommunityProposalsWithVotes.js#L7-L8) this endpoint returns up to 100 elements, then at the top level we can filter which ids we have and set the prop as admin or not.

Reply:
Made a change for this in `<CommunityProposalList />`, and realized I had been using the wrong value. Let me know what you think.